### ISSUE TITLE
SlackRequestHandler does not work on Lambda

### ISSUE THREAD

**Issue by cazziwork** [slackapi/bolt-python#132]
### Problem
I was create app by bolt for python, And deploy on Lambda.
But It's not work correct.

I investigated this issue and found problems in determining the following areas.
https://github.com/slackapi/bolt-python/blob/main/slack_bolt/adapter/aws_lambda/handler.py
line:35
`method = event.get("requestContext", {}).get("http", {}).get("method")`

On my Lambda environment, it's always return `None`.

### Workaround
I modified it below.
`method = event.get("requestContext", {}).get("httpMethod", {})`

It's work correct.

**Comment by seratch** [comment]
@cazziwork 
Thanks for sharing this! I'd love to know more details as the current implementation surely works for me and we may miss some patterns we should cover. 

Can you share more information (e.g, python runtime version you use,  code snippet, the tool you use for deployments)?

**Comment by cazziwork** [comment]
@seratch 
Thanks for your reply. I can provide information, of course.

Python Runtime: 3.8
Deploy tool: deployed zip file by browser
Lambda Handler Code Snippet:
```
import os
import sys
sys.path.insert(1, "vendor")

import logging
import secrets
import string

from slack_bolt import App
from slack_bolt.adapter.aws_lambda import SlackRequestHandler

app = App(
    signing_secret=os.environ.get("SLACK_SIGNING_SECRET"),
    token=os.environ.get("SLACK_BOT_TOKEN"),
    process_before_response=True
)

SlackRequestHandler.clear_all_log_handlers()
logging.basicConfig(format="%(asctime)s %(message)s", level=logging.DEBUG)

def lambda_handler(event, context):
    slack_handler = SlackRequestHandler(app=app)
    return slack_handler.handle(event, context)
```

**Comment by seratch** [comment]
@cazziwork Thanks for the prompt reply! In addition to that, can you share the info about your API Gateway? I haven't managed to reproduce your situation yet. If you enable a simple HTTP API as below, the `requestContext` is compatible with the current implementation of the adapter.

```
API type: HTTP
Authorization: NONE
Cross-origin resource sharing (CORS): No
Enable detailed metrics: No
Method: ANY
Resource path: /the-path-you-use
Stage: default
```

I'm happy to support the pattern you mentioned but just wanted to know what the pattern is.

**Comment by seratch** [comment]
I've checked Chalice's code and found the expected payload data structure is the same with your report. I've sent #133 to fix this and a new beta version will be released today.

**Comment by seratch** [comment]
I just released version 0.9.6b0. Try the version out when you have a chance! Thanks a lot for helping us fix this issue.

**Comment by cazziwork** [comment]
Thank you for your quick response.
I was confirmed that the issue is resolved by version 0.9.6b0:Hooray:
